modules/sno-du-applying-the-post-install-configuration-custom-resources.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: CONCEPT
```

Comment:
This also reads as a procedure. Suggest changing metadata and adding a bullet since it's only one step.